releasenotes/notes/unify-instructions-and-commands-aaa6d8724b8a29d3.yaml

```
@@ -0,0 +1,22 @@
+---
+prelude: >
+    There has been a significant simplification to the style in which Pulse
+    instructions are built.
+
+    With the previous style, ``Command`` s were called with channels to make
```

Do you want to use:
```suggestion
    With the previous style, :class:`qiskit.pulse.Command`s were called with channels to make
```
here and in other places were you reference a class?

---

Not for Command because it wants to disappear

---

Instruction should though